user/installing-dependencies.md

```diff
@@ -12,14 +12,18 @@ redirect_from:
 
 To install Ubuntu packages that are not included in the default [standard](/user/reference/precise/), use apt-get in the `before_install` step of your `.travis.yml`:
 
+By default, `apt-get update` does not get run automatically unless you are using the [APT addon](#Installing-Packages-with-the-APT-Addon). If you want to update `apt-get` automatically on every build, add something like the following to your `.travis.yml`:
```

Comment:
I believe the current logic is: 

"…you are using the [APT addon](#Installing-Packages-with-the-APT-Addon)" _or `apt-get install` is explicitly invoked in `.travis.yml`._

Reply:
And ~something like~ adds nothing here.